spec/features/user_can_log_in_and_out_spec.rb

```diff
@@ -4,14 +4,14 @@
     scenario "logs into users account" do
       user = create(:user)
 
-      visit "/"
+      visit root_path
```

Comment:
I always go back on forth on this myself, but I agree, helpers for the test navigation and urls in expectations.